### PR TITLE
Inactive stops #178

### PR DESCRIPTION
**Pull request by Antonio-John**
Filtering out inactive stops

### CODE REVIEW

File: data_transform.py [hunk 1]
```diff
@@ -129,3 +129,24 @@ def highly_serv_stops(region):
...
+    """
+    filters stops out based on the status columns. We
+    want to keep stops which are active, pending or new.
+    Parameters:
+        stops_df the dataframe wanting to filter on
+    Returns:
+        filtered_stops which meet the criteria
+        of keeping on status
+    """
+    print(len(stops))
```
Comment (jwestw) [inline data_transform.py:144]: Still got a print statement in here. Do you want this?

File: SDG_11.2.1_main.py [hunk 2]
```diff
@@ -44,8 +45,10 @@
                                                 "data",
                                                 "stops"))
 
+filtered_stops=dt.filter_stops(stops=stops_df)
```
Comment (jwestw) [inline SDG_11.2.1_main.py:48]: Nice approach!

File: data_transform.py [hunk 3]
```diff
@@ -129,3 +129,24 @@ def highly_serv_stops(region):
     """
     #  day="Wed"
     return None
+
+
+def filter_stops(stops):
+    """
+    filters stops out based on the status columns. We
```
Comment (jwestw) [inline data_transform.py:136]: Maybe change the first line to a summary:

"Filters the stops dataframe based on the status column.

....rest of the docstring..."

File: data_transform.py [hunk 4]
```diff
@@ -129,3 +129,24 @@ def highly_serv_stops(region):
...
+        filtered_stops which meet the criteria
+        of keeping on status
+    """
+    print(len(stops))
+    filtered_stops=stops[(stops["Status"]=="active") |
+                           (stops["Status"]=="pending") |
+                           (stops["Status"]==None) |
+                           (stops["Status"]=="new")]
+
+    print(len(filtered_stops))
```
Comment (jwestw) [inline data_transform.py:150]: Again, you probably don't want this print statement I am guessing.

**Review by jwestw — CHANGES_REQUESTED** [review]

File: data_transform.py (hunk 1 above)
Comment (Antonio-John) [reply data_transform.py:144]: definitely not! Will remove this.

File: data_transform.py (hunk 3 above)
Comment (Antonio-John) [reply data_transform.py:136]: yeah that makes more sense :)

**Review by jwestw — APPROVED** [review]

**Pull request merged**